docs/input.md

```diff
@@ -198,6 +223,15 @@ A material design calendar picker. Allows specifying minimum and maximum dates.
 
 Datepicker returns a **string** with the date in `YYYY-MM-DD` format.
 
+### Complex List Events that can be attached to Kiln-Input using kiln.js
+
+* **input**	- Emitted when the datepicker value is changed. The handler is called with the new value.
+* **touch** - Emitted when the datepicker is focused for the first time and then blurred.
+* **focus**	- Emitted when the datepicker is focused.
+* **blur**	- Emitted when the datepicker loses focus.
+* **open**	- Emitted when the picker (the modal or popover) is opened.
+* **close**	- Emitted when the picker (the modal or popover) is closed.
```

Comment:
Each description of the events are referencing the datepicker component, but The title is about Complex List component. Is this intended?

Reply:
👍